docs/src/pages/index.tsx

```diff
@@ -155,20 +166,105 @@ export default function Home({ location, path }: Props) {
             >
               Color is used to signal structure on a page, to highlight or emphasize...
             </Tile>
-            <Tile title="Typography" linkContent="Learn more" icon={<TypograhpyIcon />}>
+            <Tile
+              title="Typography"
+              href="/foundation/typography/"
+              linkContent="Learn more"
+              icon={<TypograhpyIcon />}
+            >
               Typography is critical for communicating the hierarchy of a page.
             </Tile>
-            <Tile title="Spacings" linkContent="Learn more" icon={<ExpandDiagonalIcon />}>
+            <Tile
+              title="Spacings"
+              href="/foundation/spacing/"
+              linkContent="Learn more"
+              icon={<ExpandDiagonalIcon />}
+            >
               Consistent spacing makes an interface more clear and easy to scan.
             </Tile>
-          </Stack>
+          </Grid>
+          <Slide maxHeight={foundationHeight} expanded={isExpandedFoundation}>
+            <div ref={foundationRef as React.RefObject<HTMLDivElement>}>
+              <Grid
+                columns="1fr"
+                rowGap="2rem"
+                columnGap="1rem"
+                tablet={{ columns: "repeat(3, 1fr)" }}
+              >
+                <Tile
+                  title="Orbit principles"
+                  href="/foundation/principles/"
+                  linkContent="Learn more"
+                  icon={<ExpandDiagonalIcon />}
```

Comment:
yep, only now I got that it supposed to be just a redirect to an overview page, but it may be not a bad solution to show it via expanded cards. Asked Will for his expert advice ;)

Reply:
okay, Will approved the expanded version, he's not sure about streamline icons, but that is what we can discuss later and solve it by separate PR

Reply:
This solution needs to be completed in this PR. If we don't have icons for these, it would be best for now to convert them to TextLinks that link to overview pages. If Will comes up with a complete solution later on, you can convert it back to Button + Slide.